src/mesh/mesh.cpp

```diff
@@ -613,23 +613,23 @@ Mesh::Mesh(ParameterInput *pin, ApplicationInput *app_in, RestartReader &rr,
   }
   EnrollBndryFncts_(app_in);
 
-  std::vector<Real> bounds = rr.ReadAttr1DReal("Mesh", "bounds");
-  mesh_size.x1min = bounds[0];
-  mesh_size.x2min = bounds[1];
-  mesh_size.x3min = bounds[2];
-  mesh_size.x1max = bounds[3];
-  mesh_size.x2max = bounds[4];
-  mesh_size.x3max = bounds[5];
+  std::vector<Real> grid_dim = rr.GetAttrVec<Real>("Info", "RootGridDomain");
```

Comment:
Can grid_dim be made const reference? At least const I think doesn't look like it is being changed.

Reply:
See previous comment.